packages/python/plotly/plotly/io/_html.py

```diff
@@ -266,12 +272,15 @@ def to_html(
         require_start = 'require(["plotly"], function(Plotly) {'
         require_end = "});"
 
-    elif include_plotlyjs == "cdn":
+    elif include_plotlyjs == "cdn" or include_plotlyjs == "cdn-latest":
+        cdn_url = plotly_cdn_url()
+        if include_plotlyjs == "cdn-latest":
+            cdn_url = plotly_cdn_url(cdn_ver="latest")
```

Comment:
Note, if this is merged after #3030 - i.e. use j2 v2, this call should be changed to:

```suggestion
            cdn_url = plotly_cdn_url(cdn_ver="latest-v2")
```

See: https://github.com/plotly/plotly.js/pull/5462